src/peer_map.rs

```diff
@@ -113,6 +113,11 @@ impl PeerMap {
     pub fn get_connection_info<'a>(&'a self, pub_id: &PublicId) -> Option<&'a ConnectionInfo> {
         self.forward.get(pub_id)
     }
+
+    // Find public id of the peer with the given socket address.
+    pub fn get_public_id<'a>(&'a self, socket_addr: &SocketAddr) -> Option<&'a PublicId> {
```

Comment:
Not used?

Reply:
Hmm... I'd expect the compiler to complain if it was unused... But maybe it doesn't because this is `pub`. I'll double-check and remove if you're right.